device-types/DataDirect Networks/SFA200NV-FC8.yaml

```diff
@@ -0,0 +1,87 @@
+---
+manufacturer: DataDirect Networks
+model: SFA200NV-FC8
+slug: sfa200nv-fc8
```

Comment:
All slugs should have the manufacturer appended. In this case, please append `datadirect-networks-`